src/main/java/tech/ada/extends_insights/domain/enums/Category.java

```diff
@@ -0,0 +1,12 @@
+package tech.ada.extends_insights.domain.enums;
+
+public enum Category {
+    SOFTWARE_DEVELOPMENT,
+    INFRASTRUCTURE_AND_NETWORKS,
+    AI_AND_MACHINE_LEARNING,
+    CYBERSECURITY,
+    DATA_SCIENCE_AND_BIG_DATA,
+    WEB_TECHNOLOGIES,
+    OS
```

Comment:
O que seria OS? Acho melhor escrevermos por extenso

Reply:
Boa, to atualizando aqui!

Reply:
atualizado